scss/civicrm/administer/display/_display.scss

```diff
@@ -73,13 +73,8 @@
         padding: 0;
       }
 
-      .crm-i {
-        margin: 0 -25px 0 15px !important;
-      }
-
       #ckeditor_config {
-        margin: 0 !important;
-        padding-left: 30px;
+        padding: 8px 12px;
```

Comment:
why did we change this code for the editor as well?

Reply:
![2020-10-20 at 12 41 PM](https://user-images.githubusercontent.com/5058867/96552589-9445cf80-12d1-11eb-8301-4235c9b9869e.png)
Its the CK Editor config button actually, which was also changed.